manim/cli/render/output_options.py

```diff
@@ -23,7 +23,7 @@
         "--write_to_movie",
         is_flag=True,
         default=None,
-        help="Write to a file.",
+        help="Write the video rendered with opengl to a file.",
```

Comment:
```suggestion
        help="Write the video rendered with Manim to a file.",
```

It's true that you only need to interact with this when using the OpenGL renderer, but it still *does* something (namely: it controls the same thing) for the Cairo renderer. My suggestion changes the wording to reflect that.